.claude/scripts/portfolio-surveyor.test.sh

```
@@ -126,10 +126,27 @@ grep -Fq '`**Actionable comments posted:`' "${surveyor}" ||
+# transient and has been measured losing a refusal (platform#3344 @ e94216b3).
 # Literal Markdown code spans; command substitution is intentionally disabled.
 # shellcheck disable=SC2016
-grep -Fq 'fails closed to `none`' "${surveyor}" ||
-  fail "surveyor does not fail closed when the head carries no CodeRabbit status"
+grep -Fq 'uninformative status' "${surveyor}" ||
+  fail "surveyor lost the uninformative-status class, so an absent or auto-review-disabled CodeRabbit status defeats a real green (#3015)"
+grep -Fq 'no CodeRabbit status at all' "${surveyor}" ||
+  fail "surveyor does not name the ABSENT-status case that must not defeat a green (#3015)"
+grep -Fq 'read a refusal from the durable' "${surveyor}" ||
+  fail "surveyor does not read a refusal from the durable reply body, so a transient status can lose it (#3015)"
+if grep -Fq 'fails closed to `none`' "${surveyor}"; then
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)